### DISCUSSION TITLE
Support using avatar from OIDC and AD/LDAP

### DISCUSSION THREAD

**Discussion by helmut72** [grafana/grafana#57881] [General]
Would be great if Grafana supports the picture URL from OIDC and if Grafana is is used with LDAP auth, Grafana should support attributes like `jpegPhoto`, `thumbnailPhoto` or `photo`.

This improves the user experience for self hosted systems without any privacy concerns. For privacy Gravatar is no option.

There was a feature request open, but the author closed it for unknown reasons. https://github.com/grafana/grafana/issues/16281
There was also a question here:
https://community.grafana.com/t/how-to-set-profile-picture-avatar-when-logging-in-with-generic-oauth-provider/50012

Thank you.

**Comment by stringang** [comment]
any update?

**Comment by latsygin** [comment]
any update?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
